example/src/shared/Wysiwyg/WysiwygNav.js

```diff
@@ -24,55 +25,107 @@ import {
-              <Option value='h6'>h6</Option>
-            </Select>
+    if (isPreviewMode) { 
+      return (
+        <NavWrapper padding={2} background='neutral100'>
+          <Row justifyContent='space-between'>
+            <Row>
+              <Select disabled id='selectTitle' placeholder={placeholder} size='S' >
+                <Option value='h1'>h1</Option>
+                <Option value='h2'>h2</Option>
+                <Option value='h3'>h3</Option>
+                <Option value='h4'>h4</Option>
+                <Option value='h5'>h5</Option>
+                <Option value='h6'>h6</Option>
+              </Select>
 
-            <MainButtons>
-              <CustomIconButton onClick={() => onActionClick("Bold")} id="Bold" label="Bold" name="Bold" icon={<Bold />} />
-              <CustomIconButton onClick={() => onActionClick("Italic")} id="Italic" label="Italic" name="Italic" icon={<Italic />} />
-              <CustomIconButton onClick={() => onActionClick("Underline")} id="Underline" label="Underline" name="Underline" icon={<Underline />} />
-            </MainButtons>
+              <MainButtons>
+                <CustomIconButton disabled id="Bold" label="Bold" name="Bold" icon={<Bold />} />
+                <CustomIconButton disabled id="Italic" label="Italic" name="Italic" icon={<Italic />} />
+                <CustomIconButton disabled id="Underline" label="Underline" name="Underline" icon={<Underline />} />
+              </MainButtons>
 
-            <IconButtonGroup>
-              <CustomIconButton onClick={() => onActionClick("Strikethrough")} id="Strikethrough" label="Strikethrough" name="Strikethrough" icon={<Strikethrough />} />
-              <CustomIconButton onClick={() => onActionClick('BulletList')} id="BulletList" label="BulletList" name="BulletList" icon={<BulletList />} />
-              <CustomIconButton onClick={() => onActionClick('NumberList')} id="NumberList" label="NumberList" name="NumberList" icon={<NumberList />} />
-            </IconButtonGroup>
-              
-            <IconButtonGroup>
-              <CustomIconButton onClick={() => onActionClick("Code")} id="Code" label="Code" name="Code" icon={<Code />} />
-              <CustomIconButton onClick={() => onActionClick("alt")} id="Image" label="Image" name="Image" icon={<Image />} />
-              <CustomIconButton onClick={() => onActionClick("Link")} id="Link" label="Link" name="Link" icon={<Link />} />
-              <CustomIconButton onClick={() => onActionClick("Quote")} id="Quote" label="Quote" name="Quote" icon={<Quote />} />
-            </IconButtonGroup>
+              <MoreButton disabled ref={buttonMoreRef} id="more" label="more" icon={<More />} />
+            </Row>
 
-            <MoreButton onClick={() => console.log('more')} id="more" label="more" icon={<More />} />
+            <Button onClick={() => setIsPreviewMode(prev => !prev)} variant='tertiary' size='L' id='preview'>
+              Markdown mode
```

Comment:
Can you add a comment so we remember to put formatMessage when migrating to code in strapi?